default.nix

```diff
@@ -119,12 +119,12 @@ rec {
       tinysparql = null;
       util-linux = super.util-linux.override { systemdSupport = false; };
       rdma-core = (optimizedBuild super.rdma-core).overrideAttrs (orig: {
-        version = "56.0";
+        version = "56.1";
         src = self.fetchFromGitHub {
-          owner = "linux-rdma";
+          owner = "githedgehog";
           repo = "rdma-core";
-          rev = "v56.0";
-          hash = "sha256-nzd7BDP72o0TsSTrCGT6HOF7td+3ex4/c68GdjIA6Bc=";
+          rev = "fix-lto-56.1";
+          hash = "sha256-nyvmDJBMPCnJP1AJw287bGFjJHiaN2kc8qvXCo/6WDg=";
```

Comment:
Out of curiosity, how do you track new upstream releases for this?

Reply:
I have it stared on github and it comes across my feed.